packages/telestion-client-cli/src/lib/modify-file.js

```diff
@@ -0,0 +1,69 @@
+const fs = require('fs');
+
+/**
+ * Insert in the line before the insertion mark
+ */
+const INSERT_ABOVE = Symbol('Insert in the line before the insertion mark');
+/**
+ * Insert in the line below the insertion mark
+ */
+const INSERT_BELOW = Symbol('Insert in the line below the insertion mark');
+/**
+ * Replace the line with the insertion mark
+ */
+const REPLACE = Symbol('Replace the line with the insertion mark');
+
+/**
+ * @typedef {Object} Replacer
+ * @property {string} needle - the string for which the file gets searched in the replacer.
+ * @property {string} text - the text that gets inserted above/below/instead of the `needle`, depending on the `position`
+ * @property {INSERT_ABOVE | INSERT_BELOW | REPLACE} position - the position where `text` gets inserted relative to the
+ * `needle`'s line.
+ */
+
+/**
+ * Modifies a file by replacing or inserting text above/below specific sections of code, identified as "needle".
+ *
+ * Processes first occurrence of each replacer, only!
+ * @param {string} filePath
+ * @param {Replacer[]} replacers
+ */
+function modifyFile(filePath, replacers) {
+	const lines = fs
+		.readFileSync(filePath, 'utf8')
+		.toString()
+		.split('\r\n')
+		.join('\n')
+		.split('\n');
+
+	for (let replacer of replacers) {
+		const insertionMarkLineIndex = lines.findIndex(line =>
+			line.includes(replacer.needle)
+		);
+
+		const newLines = [
+			lines[insertionMarkLineIndex].replace(
+				'// IMPORT_INSERT_MARK',
```

Comment:
```suggestion
				replacer.needle,
```
Some leftover constant? :smile:

Reply:
Good catch 😃